udata/core/metrics/__init__.py

```diff
@@ -111,10 +112,12 @@ def iso(self, value):
                 'Unsupported format: {0} ({1})'.format(value, type(value)))
 
     def trigger_update(self):
-        self.need_update.send(self)
+        if not current_app.config['TESTING'] or current_app.config.get('FORCE_METRICS'):
```

Comment:
Worth an entry in [settings' documentation](http://udata.readthedocs.io/en/latest/adapting-settings/)?